src/Raven.Studio/typescript/commands/database/query/queryCommand.ts

```diff
@@ -18,7 +18,7 @@ class queryCommand extends commandBase {
         const selector = (results: Raven.Client.Documents.Queries.QueryResult<Array<any>, any>): pagedResultExtended<document> =>
             ({
                 items: results.Results.map(d => new document(d)), 
-                totalResultCount: results.CappedMaxResults || results.TotalResults, 
+                totalResultCount: results.CappedMaxResults || results.LongTotalResults, 
```

Comment:
@ml054 is this ok? what will happen here if we exceed `Number.MAX_SAFE_INTEGER` ?

Reply:
We don't use it in the ace editor. So it should be fine.
@ml054

Reply:
Even if it is bigger it will be rounded, so it should be fine.